docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,1111 @@
+
+To briefly clarify the code provided above: the handler function `handleIncreaseLiquidity` is responsible for managing an increase in liquidity event on the blockchain. Initially, it invokes the `getPosition` function to either retrieve an existing liquidity position associated with the event's `tokenId` or create a new one if necessary. Any modifications made to the `Position` object during this process are saved to ensure persistence.
+
+🎉 In conclusion, we have successfully incorporated all the desired entities that can be retrieved from various smart contracts. For each of these entities, we've created mapping handlers to structure and store the data in a queryable format.
+
+::: tip Note
+Check the final code repository [here](https://github.com/subquery/ethereum-subql-starter/tree/main/Ethereum/ethereum-uniswap-v3) to observe the integration of all previously mentioned configurations into a unified codebase.
+:::
+
+## Build Your Project
+
+Next, build your work to run your new SubQuery project. Run the build command from the project's root directory as given here:
+
+::: code-tabs
+@tab:active yarn
+
+```shell
+yarn build
+```
+
+@tab npm
+
+```shell
+npm run-script build
+```
+
+:::
+
+::: warning Important
+Whenever you make changes to your mapping functions, you must rebuild your project.
+:::
+
+Now, you are ready to run your first SubQuery project. Let’s check out the process of running your project in detail.
+
+## Run Your Project Locally with Docker
+
+Whenever you create a new SubQuery Project, first, you must run it locally on your computer and test it and using Docker is the easiest and quickiest way to do this.
+
+The `docker-compose.yml` file defines all the configurations that control how a SubQuery node runs. For a new project, which you have just initialised, you won't need to change anything.
+
+However, visit the [Running SubQuery Locally](../../run_publish/run.md) to get more information on the file and the settings.
+
+Run the following command under the project directory:
+
+::: code-tabs
+@tab:active yarn
+
+```shell
+yarn start:docker
+```
+
+@tab npm
+
+```shell
+npm run-script start:docker
+```
+
+:::
+
+::: tip Note
+It may take a few minutes to download the required images and start the various nodes and Postgres databases.
+:::
+
+## Query your Project
+
+Next, let's query our project. Follow these three simple steps to query your SubQuery project:
+
+1. Open your browser and head to `http://localhost:3000`.
+
+2. You will see a GraphQL playground in the browser and the schemas which are ready to query.
+
+3. Find the _Docs_ tab on the right side of the playground which should open a documentation drawer. This documentation is automatically generated and it helps you find what entities and methods you can query.
+
+Try the following queries to understand how it works for your new SubQuery starter project. Don’t forget to learn more about the [GraphQL Query language](../../run_publish/query.md).
+
+### Pools
```

Comment:
Perhaps use the expand markdown component for each example request + response combination to save space here `::: detail xxx`

Reply:
Added as part of https://github.com/subquery/documentation/pull/415/commits/a83c0df3f088c106285121d2a0f1cec76e535045